pkg/genericapiserver/genericapiserver.go

```diff
@@ -128,6 +131,20 @@ type APIGroupVersionOverride struct {
 	ResourceOverrides map[string]bool
 }
 
+// Info about an API group.
+type APIGroupInfo struct {
+	GroupMeta latest.GroupMeta
```

Comment:
The fact that this type is in `latest` but it shows up in this class is indicative it should not be in latest - it probably belongs in the same location as GroupVersionKind (which today is unversioned, but should be a different package in the future).